pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "framegrab"
-version = "0.5.5"
+version = "0.6.0"
```

Comment:
I'm not sure updating pydantic here is a breaking change, 2.5.3 is still valid under ^2.5.3